docs/sources/configure-client/_index.md

```diff
@@ -11,9 +11,12 @@ weight: 300
 # Configure the client to send profiles
 
 Pyroscope is a continuous profiling database that allows you to analyze the performance of your applications.
-When sending profiles to Pyroscope, you can choose between two primary methods: SDK instrumentation and auto-instrumentation using Grafana Alloy.
+When sending profiles to Pyroscope, you can choose between three methods:
+1. Auto-instrumentation using Grafana Alloy
+2. SDK instrumentation
+3. SDK instrumentation through Grafana Alloy
```

Comment:
Re-reading this fresh and wondered: Should we say that the SDK instrumentation is manual process? Or do you think it's necessary? For example, should we say ""Manual instrumentation using SDKs". 

I'm not sure that we need this given the deeper explantations.

Reply:
For example: 
```suggestion
1. Auto-instrumentation using Grafana Alloy
2. Manual SDK instrumentation
3. Manual SDK instrumentation through Grafana Alloy
```

Reply:
if you like the suggestion, please feel free to edit. If you don't like it, then feel free to not use it. 

I'm making the suggestion for parallel structure reasons (first item says auto-instrumentation while the others don't specify auto or manual).

Reply:
I like it, thanks 👍